hasher-matcher-actioner/hmalib/lambdas/pdq/pdq_matcher.py

```diff
@@ -120,7 +120,7 @@ def lambda_handler(event, context):
                 # to dataset[s]
                 for privacy_group in metadata.get("privacy_groups", []):
                     banked_signal = BankedSignal(
-                        signal_id, privacy_group, metadata["source"]
+                        str(signal_id), str(privacy_group), str(metadata["source"])
```

Comment:
Ran into an issue where `signal_id` is an int, and its intiness was causing an issue later down the line. (An equality check between, say 12345 and "12345" was failing.